README.md

```diff
@@ -101,7 +101,7 @@ aelf.chain.contractAtAsync(contractAddress, wallet, (err, result) => {
 });
 ```
 
-Additionally you can set a provider using aelf.setProvider()
+Additionally, you can set a provider using aelf.setProvider()
```

Comment:
https://english.stackexchange.com/questions/21532/when-to-use-commas-in-a-sentence-that-starts-with-finally-additionally-etc